src/rev_ai/apiclient.py

```diff
@@ -0,0 +1,139 @@
+        """Submit media given a URL for transcription.
+        The audio data is downloaded from the URL.
+
+        :param media_url: web location of the media file
+        :param options: JobSubmitOptions object for the job
+        :returns: raw response data
+        """
+        url_jobs = urljoin(self.base_url, "jobs")
+        payload = {
+            'media_url': media_url,
+            'metadata': options.metadata
+        }
+        if options.callback_url:
+            payload['callback_url'] = options.callback_url
+        response = self.s.post(url_jobs, json=payload)
+
+        return Job.from_json(response.json())
+
+    def submit_job_local_file(self, filename, options):
+        """Submit a local file for transcription.
+        Note that the content type is inferred if not provided.
+
+        :param filename: path to a local file on disk
+        :param content_type: MIME content type of file being sent
+        :param options: JobSubmitOptions object for the job
+        :returns: raw response data
+        """
+        url_jobs = urljoin(self.base_url, "jobs")
+        payload = {'metadata': options.metadata}
+        if options.callback_url:
+            payload['callback_url'] = options.callback_url
+
+        with open(filename, 'rb') as f:
+            files = {
+                'media': (filename, f),
+                'options': (None, json.dumps(payload))
+            }
+            response = self.s.post(url_jobs, files=files)
+
+        return Job.from_json(response.json())
+
+    def get_job_details(self, id_):
+        """View information about a specific job.
+        The server will respond with the status and creation date.
+
+        :param id_: id that the server gave you
+        :returns: raw response data
+        """
+        url_jobs_id = urljoin(self.base_url, "jobs/{id_}".format(id_=id_))
+
+        response = self.s.get(url_jobs_id)
+
+        return Job.from_json(response.json())
+
+    def get_transcript_text(self, id_):
+        """Get the transcript of a specific job as json.
+
+        :param id_: id that the server gave you
+        :returns: transcript data as text
+        """
+        url_jobs_transcript = urljoin(
+            self.base_url,
+            "jobs/{id_}/transcript".format(id_=id_)
+        )
+
+        response = self.s.get(
+            url_jobs_transcript,
+            headers={'Accept': "text/plain"}
+        )
+
+        return response.text
+
+    def get_transcript_object(self, id_, format_):
+        """Get the transcript of a specific job as json.
+
+        :param id_: id that the server gave you
+        :returns: transcript data as a python object
+        """
+
+        url_jobs_transcript = urljoin(
+            self.base_url,
+            "jobs/{id_}/transcript".format(id_=id_)
+        )
+
+        response = self.s.get(
+            url_jobs_transcript,
+            headers={'Accept': 'application/{version}+json'
```

Comment:
Is there a naming collision with the version defined above?
https://github.com/revdotcom/revai-python-sdk/pull/3/files#diff-e8a9624e829d294d93268cd2b25fb8f2R20

We should rename to clarify that this is the transcript version and not the api version.

Reply:
no there isn't, that one is namespaced under `s`, via `s.version`. `version` here is the name of the parameter to `str.format`.

but I could see how this looks odd